toxav/bwcontroller.h

```diff
@@ -6,12 +6,18 @@
 #define C_TOXCORE_TOXAV_BWCONTROLLER_H
 
 #include "../toxcore/Messenger.h"
+#include "../toxcore/tox.h"
```

Comment:
This requires "//c-toxcore/toxcore" to be added to the deps of the bwcontroller cc_library in BUILD.bazel.

Reply:
done